src/main/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStatic.java

```diff
@@ -270,4 +273,43 @@ private static boolean isMethodDeclarationWithAnnotation(@Nullable Statement sta
         }
         return false;
     }
+
+    private static J.@Nullable Identifier findMockedStaticVariable(Cursor scope, String className) {
+        JavaSourceFile compilationUnit = scope.firstEnclosing(JavaSourceFile.class);
+        if (compilationUnit == null) {
+            throw new IllegalStateException("A JavaSourceFile is required in the cursor path.");
```

Comment:
Can we safely return `null` here instead? I don't think it'll ever happen, but it's still not a pattern we want to have folks think is ok.